### PR TITLE
create Navbar and add protected routes

### PR DESCRIPTION
**Pull request by Emumba-Abdullah**

### CODE REVIEW

File: frontend/src/Router.tsx [hunk 1]
```diff
@@ -31,6 +32,10 @@ const router = createBrowserRouter([
         path: "/AddTour",
         element: <AddTour />,
       },
+      {
+        path: "/Tours",
```
Comment (AliZaib-Emumba) [inline frontend/src/Router.tsx]: paths usually start with lowercase. Please make sure all of your routes are lowercased.

File: frontend/src/components/NavBar.tsx [hunk 2]
```diff
@@ -7,18 +7,24 @@ import Typography from '@mui/material/Typography';
...
 import MenuIcon from '@mui/icons-material/Menu';
 import Container from '@mui/material/Container';
-import Avatar from '@mui/material/Avatar';
 import Button from '@mui/material/Button';
 import Tooltip from '@mui/material/Tooltip';
 import MenuItem from '@mui/material/MenuItem';
 import logo from './../assets/logo.png';
+import AccountCircleRoundedIcon from '@mui/icons-material/AccountCircleRounded';
+import { NavLink } from "react-router-dom"; 
+const pages = ['Tours', 'Add Tour', 'My Tours'];
```
Comment (AliZaib-Emumba) [inline frontend/src/components/NavBar.tsx]: Other code definitions and variables declaration must be below the library imports.

Also, Try to make library imports separate from directory imports. E:g., 
```
     import Menu from '@mui/material/Menu';
     import MenuIcon from '@mui/icons-material/Menu';
     
     import logo from './../assets/logo.png';
```

File: frontend/src/components/NavBar.tsx [hunk 3]
```diff
@@ -80,24 +95,30 @@ export default function NavBar() {
             </Menu>
           </Box>
           <Box sx={{ flexGrow:1,display: { xs: 'flex', md: 'none' }, mr: 1, }}>
-             <img src={logo} alt="Kitty Katty!" style={{maxWidth:'300px', maxHeight:'40px'}} />
+             <img src={logo} alt="TourBay" style={{maxWidth:'300px', maxHeight:'40px'}} />
           </Box>
           <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' }, marginRight:10}}>
             {pages.map((page) => (
               <Button
```
Comment (AliZaib-Emumba) [inline frontend/src/components/NavBar.tsx]: Button component is not required here.

File: frontend/src/components/NavBar.tsx [hunk 4]
```diff
@@ -80,24 +95,30 @@ export default function NavBar() {
...
+               className={({ isActive, isPending }) =>
+               isPending ? "pending" : isActive ? "active" : ""
+               }>
+                  {page}
+                </NavLink>
               </Button>
             ))}
           </Box>
 
           <Box sx={{ flexGrow: 0 }}>
```
Comment (AliZaib-Emumba) [inline frontend/src/components/NavBar.tsx]: default values should not be mentioned.

File: frontend/src/components/NavBar.tsx [hunk 5]
```diff
@@ -116,11 +137,11 @@ export default function NavBar() {
               open={Boolean(anchorElUser)}
               onClose={handleCloseUserMenu}
             >
-              {settings.map((setting) => (
```
Comment (AliZaib-Emumba) [inline frontend/src/components/NavBar.tsx:119]: Do you think keepMounted property should be used here ?

Comment (Emumba-Abdullah) [reply frontend/src/components/NavBar.tsx:119]: it is used for the intend to enhance user experience by maintaining the menu's state between open and close actions. If we want the menu to stay mounted even when it's closed, we should include the keepMounted prop. I used MUI component and searched about it..

**Pull request closed without merging**